tests/antithesis/test-template/robustness/main.go

```diff
@@ -48,26 +49,41 @@ const (
 	defaultetcd1 = "etcd1:2379"
 	defaultetcd2 = "etcd0:2379"
 
-	localetcd0 = "127.0.0.1:12379"
-	localetcd1 = "127.0.0.1:22379"
-	localetcd2 = "127.0.0.1:32379"
+	localetcd0           = "127.0.0.1:12379"
+	localetcd1           = "127.0.0.1:22379"
+	localetcd2           = "127.0.0.1:32379"
+	defaultEtcdDataPath  = "/var/etcd%d/data"
+	localEtcdDataPathEnv = "ETCD_ROBUSTNESS_DATA_PATH"
 )
 
 func main() {
 	local := flag.Bool("local", false, "run tests locally and connect to etcd instances via localhost")
 	flag.Parse()
+	persistedRequestdirs := []string{
+		fmt.Sprintf(defaultEtcdDataPath, 0),
+		fmt.Sprintf(defaultEtcdDataPath, 1),
+		fmt.Sprintf(defaultEtcdDataPath, 2),
+	}
 	hosts := []string{defaultetcd0, defaultetcd1, defaultetcd2}
 	if *local {
 		hosts = []string{localetcd0, localetcd1, localetcd2}
+		localpath := os.Getenv(localEtcdDataPathEnv)
+		if localpath != "" {
```

Comment:
Why not default to `/tmp/etcddata`?

Reply:
That was poor naming from me.  How about now?

Reply:
LG